war/src/main/scss/modules/notifications.scss

```diff
@@ -3,7 +3,7 @@
   left: 1.2rem;
   bottom: 1.2rem;
   min-width: 321px;
-  max-width: ~"min(600px, calc(100vw - 2.4rem))";
+  max-width: min(600px, calc(100vw - 2.4rem));
```

Comment:
This produces a difference in the compiled CSS: from `max-width: min(600px, calc(100vw - 2.4rem));` to `max-width: min(600px, 100vw - 2.4rem);`. Is that intentional? If not, please disregard this comment.

Reply:
Fixed :)